services/121-service/migration/1714636201547-remove-app-props-from-instance.ts

```diff
@@ -0,0 +1,31 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class RemoveAppPropsFromInstance1714636201547
+  implements MigrationInterface
+{
+  name = 'RemoveAppPropsFromInstance1714636201547';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "121-service"."instance" DROP COLUMN "logoUrl"`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "121-service"."instance" DROP COLUMN "dataPolicy"`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "121-service"."instance" DROP COLUMN "contactDetails"`,
+    );
```

Comment:
You could do it by a single query like 😊 
```
ALTER TABLE "121-service"."instance"
	DROP COLUMN "logoUrl",
	DROP COLUMN "dataPolicy",
	DROP COLUMN "contactDetails";
```

It's not a blocker, approving the MR!

Reply:
Nice one, updated it :)